docs/topics/db/models.txt

```diff
@@ -1238,6 +1238,11 @@ number of proxy models that share a common non-abstract parent class.
     In earlier versions, a proxy model couldn't inherit more than one proxy
     model that shared the same parent class.
 
+.. note::
```

Comment:
Don't use "note" use "admonition" with a descriptive title (if a call out box is even needed).